src/main/java/io/github/bobocodebreskul/server/TomcatServer.java

```diff
@@ -39,7 +40,7 @@ public static void run(BringContainer container) {
     tomcat = new Tomcat();
     tomcat.setHostname(DEFAULT_HOST);
     tomcat.getHost().setAppBase(DOC_BASE);
-    tomcat.setPort(DEFAULT_PORT);
+    tomcat.setPort(PropertiesConfiguration.getPropertyAsIntOrDefault("port", DEFAULT_PORT));
```

Comment:
I would declare 'port' as a constant variable and write a short javadoc that explains the purpose of this variable.
Additionally, it would be nice give more specific  name for this property 'server.port', so that all internal application configurations made by end user from .properties file would start from 'server'.

Reply:
i agree, and maybe lets do method call in separate line, it will do code more readable